images/ceph/Makefile

```diff
@@ -122,13 +127,11 @@ generate-csv-templates: $(OPERATOR_SDK) $(YQv3) ## Generate CSV templates for OL
 	@echo " === Generated CSV templates can be found at $(CSV_TEMPLATE_DIR)/cluster/olm/ceph/templates"
 
 get-volume-replication-crds:
-	@if [[ -z "$(CRD_TEMPLATE_DIR)" ]]; then echo "CRD_TEMPLATE_DIR is not set"; exit 1; fi
-	@if [[ ! -d "$(CACHE_DIR)/crds" ]]; then\
+	if [[ ! -d "$(CACHE_DIR)/crds" ]]; then\
 		mkdir -p $(CACHE_DIR)/crds;\
 		curl -L $(VOL_REPL_URL)/$(VOLUME_REPLICATION_CRD) -o $(CACHE_DIR)/crds/$(VOLUME_REPLICATION_CRD);\
 		curl -L $(VOL_REPL_URL)/$(VOLUME_REPLICATION_CLASS_CRD) -o $(CACHE_DIR)/crds/$(VOLUME_REPLICATION_CLASS_CRD);\
 	fi
-	@cp $(CACHE_DIR)/crds/* $(CRD_TEMPLATE_DIR)
```

Comment:
In order to download the files to be staged for later steps, we don't actually copy the files into anywhere. Just want to download them, so I separated the get/download stage from the copy stage.